tests/address_bar_and_search/test_adaptive_history_autofill.py

```diff
@@ -34,7 +33,7 @@ def test_add_adaptive_history_autofill(driver: Firefox):
     driver.switch_to.window(driver.window_handles[1])
 
     with driver.context(driver.CONTEXT_CHROME):
-        x_icon = tabs.get_element("tab-x-icon", multiple=True)
```

Comment:
the `multiple` flag still exists on `get_element` but instances of it were replaced with its sister method that does the same thing. Should that flag be removed?